### PR TITLE
Configure https by devcert

### PR DESCRIPTION
**Pull request by dyoshikawa**
As a LIFF developer, I am looking for an easy way to verify access and ID tokens in a local environment. I find deploying this to Nelilty to be too much of a hassle. The LIFF platform requires endpoints to be https, so the local environment should adapt to that. I would like to be able to simply run a git clone of this liff-playground and run npm start to check its values.

So I have made it possible to do https communication in my local environment. I believe this change will be in great demand by many LIFF developers and people who want to try LIFF.

## Usage

1. Set `https://localhost:3000` to the endpoint of our LIFF app.
2. Run that commands:

```bash
npm i
npm start
```

The screenshot in my local machine:
Screen Shot 2022-05-11 at 23 29 15

### CODE REVIEW

**Comment by LeeChSien** [comment]
Thanks for your contribution.  Once you finish the implementation, please describe the detail for this PR and set the status as ready for review.

**Comment by dyoshikawa** [comment]
@LeeChSien Thank you. Now implementation is complete, please review.

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "typescript": "4.4.3"
   },
   "scripts": {
-    "start": "craco start",
+    "start": "HTTPS=true craco start",
```
Comment (LeeChSien) [inline package.json]: Thanks, that's a good idea to use `https` in local, could you create a new script `start:https`, and add the usage in README.md?

BTW, we are going to refactor this repo with `create-liff-app`.
https://github.com/line/liff-playground/pull/8
I will keep this script and migrate it to `vite`.

**Comment by dyoshikawa** [comment]
Let this PR take over.
https://github.com/line/liff-playground/pull/10

**Pull request closed without merging**